sdk/cosmos/src/operations/get_document.rs

```diff
@@ -1,38 +1,85 @@
 use crate::headers::from_headers::*;
+use crate::prelude::*;
 use crate::resources::Document;
 use crate::ResourceQuota;
 use azure_core::headers::{etag_from_headers, session_token_from_headers};
+use azure_core::prelude::*;
 use azure_core::SessionToken;
+use azure_core::{collect_pinned_stream, Request as HttpRequest, Response as HttpResponse};
 use chrono::{DateTime, Utc};
-use http::response::Response;
-use http::StatusCode;
+use http::{HeaderMap, StatusCode};
 use serde::de::DeserializeOwned;
 
+#[derive(Debug, Clone)]
+pub struct GetDocumentOptions<'a, 'b> {
+    document_client: &'a DocumentClient,
```

Comment:
We don't need the document client or activity id in the options. They can just be removed.

Reply:
Ah, thanks. It wasn't clear to me which things were needed and which weren't. I've removed the two now.